openapi.json

```diff
@@ -1380,6 +1380,61 @@
           "prod"
         ]
       }
+    },
+    "/overview": {
```

Comment:
Maybe "org_overview" or something like this?

Reply:
Good idea